### DISCUSSION TITLE
关于一直安装不成功的问题

### DISCUSSION THREAD

**Discussion by tsuyoshikate** [babysor/MockingBird#745] [Q&A]
preparing metadata (pyproject.toml)这部分一直无法下载，导致无法运行，缺少文件。
请问大大，这个问题如何解决？谢谢！

**Comment by wwt0805** [comment]
> preparing metadata (pyproject.toml)这部分一直无法下载，导致无法运行，缺少文件。 请问大大，这个问题如何解决？谢谢！

我也是metadata报错，请问解决了么
